Tasks/AzureResourceManagerTemplateDeploymentV3/operations/ResourceGroup.ts

```diff
@@ -0,0 +1,67 @@
+import tl = require("azure-pipelines-task-lib/task");
+
+import deployAzureRG = require("../models/DeployAzureRG");
+import armResource = require("azure-arm-rest-v2/azure-arm-resource");
+import utils = require("./Utils");
+import { DeploymentScopeBase } from "./DeploymentScopeBase";
+
+export class ResourceGroup extends DeploymentScopeBase {
+
+    public resourceManagementClient: armResource.ResourceManagementClient;
+
+    constructor(resourceManagementClient: armResource.ResourceManagementClient, taskParameters: deployAzureRG.AzureRGTaskParameters) {
+        super(resourceManagementClient, taskParameters);
+        this.resourceManagementClient = resourceManagementClient;
+    }
+
+    public async createOrUpdateResourceGroup(): Promise<void> {
```

Comment:
you can call this function `deploy` itself. it can override the default implementation of base

Reply:
Completed this change